libs/vkd3d/device_vkd3d_ext.c

```diff
@@ -129,6 +130,34 @@ static HRESULT STDMETHODCALLTYPE d3d12_device_vkd3d_ext_CreateCubinComputeShader
     return S_OK;
 }
 
+static HRESULT STDMETHODCALLTYPE d3d12_device_vkd3d_ext_CreateCubinComputeShaderWithName(ID3D12DeviceExt *iface, const void *cubin_data,
+       UINT32 cubin_size, UINT32 block_x, UINT32 block_y, UINT32 block_z, const char *shader_name, D3D12_CUBIN_DATA_HANDLE **out_handle)
+{
+    return d3d12_device_vkd3d_ext_CreateCubinComputeShader_common(iface,
+                                                                  cubin_data,
+                                                                  cubin_size,
+                                                                  block_x,
+                                                                  block_y,
+                                                                  block_z,
+                                                                  0, /* smem_size */
+                                                                  shader_name,
+                                                                  out_handle);
+}
+
+static HRESULT STDMETHODCALLTYPE d3d12_device_vkd3d_ext_CreateCubinComputeShaderEx(ID3D12DeviceExt1 *iface, const void *cubin_data,
+       UINT32 cubin_size, UINT32 block_x, UINT32 block_y, UINT32 block_z, UINT32 smem_size, const char *shader_name, D3D12_CUBIN_DATA_HANDLE **out_handle)
+{
+    return d3d12_device_vkd3d_ext_CreateCubinComputeShader_common(iface,
```

Comment:
Lots of pointer type warnings here with the ID3D12DeviceExt interface.

Reply:
Whoops, have this resolved locally with a pre-commit. Looks like the original changes never really made use of the `d3d12_device_vkd3d_ext_iface` typedef that they added.